MANIFEST.in

```diff
@@ -0,0 +1,3 @@
+recursive-exclude discogs_client/tests *symlinks*
```

Comment:
Could you make this reference the exact file?
i.e., `exclude discogs_client/tests/make_symlinks.py`